src/main/java/com/box/sdk/BoxZip.java

```diff
@@ -0,0 +1,118 @@
+package com.box.sdk;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URL;
+import java.util.List;
+
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+
+/**
+ * Provides methods to allow users to download multiple files and folders as a single zip file. Users can download
+ * up to either 32GB or 10,000 files in one batch (whichever limitation is hit first) as a single zip file.
+ */
+public class BoxZip {
+    /**
+     * Zip URL Template.
+     */
+    public static final URLTemplate ZIP_URL_TEMPLATE = new URLTemplate("zip_downloads");
+    /**
+     * Zip Download URL Template.
+     */
+    public static final URLTemplate ZIP_DOWNLOAD_URL_TEMPLATE = new URLTemplate("zip_downloads/%s/content");
+    private static final int BUFFER_SIZE = 8192;
+    private final BoxAPIConnection api;
+
+    /**
+     * Constructs a Zip to be used by everything.
+     *
+     * @param api the API connection to be used by the Zip.
+     */
+    public BoxZip(BoxAPIConnection api) {
+        this.api = api;
+    }
+
+    /**
+     * Creates a zip of multiple files and folders.
+     *
+     * @param name  the name of the zip file to be created
+     * @param items list of files or folders to be part of the created zip
+     * @return      information about the created zip file
+     */
+    public BoxZipInfo create(String name, List<JsonObject> items) {
```

Comment:
Can we create a new model for this parameter instead of using JsonObject?  Obviously it would only need type and id, but it might make things more clear.